chronograf.go

```diff
@@ -565,16 +565,22 @@ type Dashboard struct {
 	Name      string          `json:"name"`
 }
 
+// DashboardRange represents the visible extents of a visualization
+type DashboardRange struct {
```

Comment:
This type should be Axis.

It'll include bounds, but, in the future it'll include other properties of the axis.